CONTRIBUTING.md

```diff
@@ -152,3 +152,15 @@ rm -rf IHP
 rm build/ihp-lib
 ./start
 ```
+
+## Resources
+
+Practical resources to get you up to speed with tools IHP uses.
+
+### Haskell
+
+-   [What I Wish I Knew When Learning Haskell](http://dev.stephendiehl.com/hask/)
```

Comment:
I'd add https://www.haskell.org/hugs/pages/users_guide/implicit-parameters.html :)

Reply:
I've added a link to the new GHC user guide, it seems to be the same content, just easier to read with syntax highlighting.